openslide/deepzoom.py

```diff
@@ -208,42 +218,62 @@ def _get_tile_info(self, dz_level, t_location):
         )
 
         # Get final size of the tile
-        z_size = tuple(
-            min(self._z_t_downsample, z_lim - self._z_t_downsample * t) + z_tl + z_br
-            for t, z_lim, z_tl, z_br in zip(
-                t_location, self._z_dimensions[dz_level], z_overlap_tl, z_overlap_br
+        z_size = (
```

Comment:
the changes over the next few lines are to allow for the returned tuples to be 2-tuple rather than n-tuples where appropriate to fit with other calls e.g read_region

Reply:
It turns out we can just e.g. `assert len(z_size) == 2` to narrow the type to `tuple[int, int]`, so I've reverted these changes.